src/electron.js

```diff
@@ -82,7 +83,7 @@ function createWindow() {
   mainWindow.loadURL(startUrl);
 
   // Emitted when the window is closed.
-  mainWindow.on('closed', function() {
+  mainWindow.on('closed', function () {
```

Comment:
Was this committed with `--no-verify`? Our prettier settings should strip spaces like this (just tested locally), and we have a pre-commit hook to run Prettier.

I personally don't have a preference for spacing like this, just wanna make sure Prettier is working consistently for everyone to avoid a bunch of noise in future PRs.

Reply:
I'll clean this up. Flow + Windows antivirus was making my machine hang so I had to drop the precommit

Reply:
No problem at all =) just wanted to make sure there wasn't an issue with our code formatting tools.